use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -340,54 +341,68 @@ main(int argc, char *argv[])
             // misleading. It needs to be hard-coded to a literal string at
             // compile-time. The metric's name given below is the right one
             // corresponding to the active clock.
-            size_t throughput = (uint64_t)
-                        ((clientp->num_ops * 1.0 / clientp->cumu_time_ns)
-                                    * L3_NS_IN_SEC);
+            size_t throughput = 0;
 
             printf("Server: Client ID=%d exited. num_ops=%" PRIu64 " (%s)"
-                   ", cumu_time_ns=%" PRIu64
+                   ", cumu_time_ns=%" PRIu64 " (%s ns)"
                    " (clock_id=%d)"
                    ", Avg. %s time=%" PRIu64 " ns/msg"
                    ", Server-throughput=%lu (%s) ops/sec"
                    ", # active clients=%d\n",
                    req.clientId,
                    clientp->num_ops, value_str(clientp->num_ops),
-                   clientp->cumu_time_ns,
+                   clientp->cumu_time_ns, value_str(clientp->cumu_time_ns),
                    clock_id, time_metric_name(clock_id),
                    (clientp->cumu_time_ns / clientp->num_ops),
                    throughput, value_str(throughput),
                    NumActiveClients);
 
-            if (NumActiveClients == 0) {
-                break;
-            }
             // Client has exited, so no need to send ack-response back to it.
             req.clientId = 0;
+            if (NumActiveClients == 0) {
+                goto end_forever_loop;
```

Comment:
This fixes a bug in previous version where, under some cases, we would be [incorrectly] doing a `break` here. This trips up a latter assertion checking for `#-of-clients-left == 0`.

I've seen this assertion trip-up in my own dev-runs, and this fix has remedied that problem.

CI-jobs that were previously failing (in `/main` also) seems to be succeeding well now.